src/Adliance.AzureTools/CopyDatabase/Parameters/CopyDatabaseParameters.cs

```diff
@@ -5,8 +5,8 @@ namespace Adliance.AzureTools.CopyDatabase.Parameters
     [Verb("copy-database", HelpText = "Copies an existing database (structure and data) to another one. If the target database already exists, it will be overwritten. Please note that this uses SqlPackage and currently only works on Windows.")]
     public class CopyDatabaseParameters
```

Comment:
Could we please add a parameter that overrides the Azure check? Because we need to use the tool in scripted environments as well that automatically copy databases between Azure instances, and we can't use anything "interactive" there. I think it's a good idea to have the warning by default, but there should be the option to automatically accept it.